### PR TITLE
Copter: change description of system id delay variable (NFC)

### PR DESCRIPTION
**Pull request by esaldiran**
Current description of SYSTEM_ID_DELAY variable is taken from mode_poshold.cpp file. This pr changes parameter description to match with its actual functionality.

### CODE REVIEW

**Comment by rmackay9** [comment]
@lthall what do you think about the comment change?  I think what's there looks out-of-date but your choice on what the comment should be.

**Review by peterbarker — APPROVED** [review]

**Comment by lthall** [comment]
Sometimes all I can do is shake my head when I look at my code..... You can see that I must have copied from PosHold and then missed the comment change.

Yes, this is much more appropriate.

File: ArduCopter/mode_systemid.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ ModeSystemId::ModeSystemId(void) : Mode()
     AP_Param::setup_object_defaults(this, var_info);
 }
 
-#define SYSTEM_ID_DELAY     1.0f      // speed below which it is always safe to switch to loiter
+#define SYSTEM_ID_DELAY     1.0f      // time in second waited after system id mode change for frequency sweep injection
```
Comment (lthall) [inline ArduCopter/mode_systemid.cpp:72]: Just change second to seconds and this comment is perfect.

**Comment by rmackay9** [comment]
@esaldiran,

Txs for this.  Could you incorporate the change from lthall?  Then we can merge, thanks!

File: ArduCopter/mode_systemid.cpp (hunk 1 above)
Comment (esaldiran) [reply ArduCopter/mode_systemid.cpp]: Thanks @lthall, fixed the typo.

**Comment by esaldiran** [comment]
@rmackay9 I incorporated the change. Thanks.

**Comment by rmackay9** [comment]
CI was failing with the intelligent energy driver again but clearly can't be related so merged anyway, thanks!

**Pull request merged**